cmd/mattermost/commands/root.go

```diff
@@ -25,4 +26,8 @@ func init() {
 	RootCmd.PersistentFlags().Bool("disableconfigwatch", false, "When set config.json will not be loaded from disk when the file is changed.")
 	RootCmd.PersistentFlags().Bool("platform", false, "This flag signifies that the user tried to start the command from the platform binary, so we can log a mssage")
 	RootCmd.PersistentFlags().MarkHidden("platform")
+
+	viper.SetEnvPrefix("mm")
```

Comment:
Leverage the global `viper` instance. This doesn't conflict with the viper instance created in the `config` package.